helm-chart/templates/_helpers.tpl

```diff
@@ -119,6 +119,10 @@ build it here and use it everywhere.
   value: {{ .Values.brokers.gcnClassicOverKafka.groupId | quote }}
 - name: HOPSKOTCH_GROUP_ID
   value: {{ .Values.brokers.hopskotch.groupId | quote }}
+- name: HERMES_EMAIL
```

Comment:
Please handle in a way consistent with `HERMES_EMAIL_PASSWORD` (see other comments)